packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -428,6 +430,14 @@ class GestureDetector extends StatelessWidget {
   ///  * [kTertiaryButton], the button this callback responds to.
   final GestureTapCancelCallback onTertiaryTapCancel;
 
+  /// A pointer that might cause a double tap has contacted the screen at a
+  /// particular location.
+  ///
```

Comment:
Can you add some more information here about when this will fire? e.g. after the down even of the second tap? May also be helpful to put this in relation to onTapDown (presumably, that one will fire for the first tap down - does it also fire for the second down?)

Reply:
Done.